cw_monero/lib/transaction_history.dart

```diff
@@ -102,6 +106,59 @@ PendingTransactionDescription createTransactionSync(
       pointerAddress: pendingTransactionRawPointer.address);
 }
 
+PendingTransactionDescription createTransactionMultDestSync(
+    {List<String> addresses,
+      String paymentId,
+      List<String> amounts,
+      int size,
```

Comment:
we need single structure for address + amount and remove size argument, because in case if we would to have list of MoneroOutput(maybe ?) class here we doesn't need to rely on unconsistent ""size" argument.